tests/lib/rules/no-reassign-constant.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @fileoverview Tests for no-constant-reassigment rule.
+ * @author Jason Brumwell
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var eslint = require("../../../lib/eslint"),
+    ESLintTester = require("eslint-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+var eslintTester = new ESLintTester(eslint);
+eslintTester.addRuleTest("lib/rules/no-reassign-constant", {
+    valid: [
+        {code: "const a = 'a';", ecmaFeatures: { blockBindings: true }},
```

Comment:
You should add tests using `let` and `var` to ensure you're only flagging `const`